### ISSUE TITLE
Poor documentation

### ISSUE THREAD

**Issue by ghost** [bitcoin-abe/bitcoin-abe#146]
Hi. First of all thanks a lot for this nice block explorer. I am using it privately for some time now.

I don't know if you guys have time for this, but I think the documentation is really, really, really poor. There are old configs all over the place and someone new trying to get into bitcoin abe has a hard time figuring out what to do.

I'd help, but although I have abe running for several coins I'm still not sure about all the config options, new policies etc.

Can you please make bitcoin-abe more accessible to everybody? If someone has some spare time who knows the codebase very well, please write up some easy to understand tutorials. How to set up bitcoin, litecoin, somecoin with bitcoin abe.

**Comment by boombaklot** [comment]
Yes i would also like some more info how to setup configs. Im new to bitcoin abe but ive been usings parsers and others crypto tools for x11 coins and im confused. If not i can always search around online no biggie
